### PR TITLE
Support and software filters

### PR DESCRIPTION
**Pull request by galund**
Filters and indexing for G9 Cloud Software and Cloud Support lots.

https://trello.com/c/3XUKd0tX/345-add-filters-to-search-results-page

### CODE REVIEW

**Comment by samuelhwilliams** [comment]
Can we squash these commits a little please... :)

File: frameworks/g-cloud-9/questions/services/freeVersionTrialOption.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Trial option
 question: Do you provide a free trial option for your service?
-
+filter_label: free trial
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/freeVersionTrialOption.yml]: Can we keep a linebreak after the filter_label so they stay in four separate blocks, cheers... :pedant:

File: frameworks/g-cloud-9/questions/services/publicSectorNetworksTypes.yml [hunk 2]
```diff
@@ -15,13 +15,19 @@ type: checkboxes
 options:
   - label: Public Services Network (PSN)
     value: psn
+    filter_label: Public Services Network (PSN)
```
Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/publicSectorNetworksTypes.yml]: Guess it doesn't matter now, but we don't really need to add filter_labels if they are the same as the existing label. Would be nice to only override if it there's a difference.

Comment (karlchillmaid) [inline frameworks/g-cloud-9/questions/services/publicSectorNetworksTypes.yml]: I'm going to remove the ones like this where they are exactly the same. Am going to leave lower-case variants for now, though.

**Pull request merged**